test/macros.jl

```diff
@@ -265,6 +265,32 @@ function macros_test(ModelType::Type{<:JuMP.AbstractModel}, VariableRefType::Typ
         @test con.set == MOI.SecondOrderCone(2)
     end
 
+    @testset "@build_constraint (SOS1)" begin 
+        model = ModelType() 
+        @variable(model, x[1:3]) 
+        con = @build_constraint(x in JuMP.SOS1()) 
+        @test con isa JuMP.VectorConstraint 
+        @test con.func == x 
+        @test con.set == MOI.SOS1([1.0, 2.0, 3.0])
```

Comment:
Looks like you also need to test:
```Julia
con2 = @build_constraint(x in JuMP.SOS1([1, 3, 2])
@test con2.func == x
@test con2.set == MOI.SOS1([1.0, 3.0, 2.0])
```
And ditto for SOS2.

You can see which lines aren't covered by looking at the coverage report:
https://codecov.io/gh/JuliaOpt/JuMP.jl/compare/2532e46ad79bebc75407a82a69e9ee11bec5568b...96bfa1610683131c71933d8c4bbdbdd32713a03c/diff

Reply:
Okay! Thank you, just added it